### ISSUE TITLE
Call out Appium compatibility

### ISSUE THREAD

**Issue by paul-hammant** [nativescript-vue/nativescript-vue#256]
### What problem does this feature solve?
Selenium/WebDriver rule for in-browser UI clicking test automaton. Appium is the standard for native apps. What's the recommended way for nativescript-vue?  Is there a link to a self contained example project?

### What does the proposed API look like?
n/a

**Comment by rigor789** [comment]
Appium works with NativeScript-Vue, but it is not currently documented, nor there is an official example project.

One project that uses it: https://github.com/nativescript-vue/nativescript-vue-ui-tests
It's used to generate the screenshots for the docs.

**Comment by msaelices** [comment]
@rigor789 what do you think is the better way to solve this issue?

BTW, `nativescript-vue-ui-tests` app is based on NS-Vue 1.0.0. Do you think is worthy to upgrade it to `2.0.0-alpha.1`

**Comment by rigor789** [comment]
@msaelices yes definitely worth upgrading, and making sure that it still works. The current screenshots for the docs have been generated by Eddy if I remember correctly, eventually I would like to enable demo mode on android (sets time and freezes it, and hides notifications) to make the screenshots even nicer.

As for solving the issue, we really need to explore testing as a whole for NativeScript-Vue, which is on the agenda. Working towards enabling appium tests for the core framework for reporoducing common issues, and automating the tests will get us closer to the goal - and then we just need to document it.

**Comment by msaelices** [comment]
@rigor789 the `nativescript-vue-ui-tests` repo is now working with the latest NS, NS-vue and Appium versions. You only need to patch the `node_modules/nativescript-dev-appium/lib/capabilities-helper.js` until [the PR I made](https://github.com/NativeScript/nativescript-dev-appium/pull/141) was accepted. See the PR in order to reproduce the change inside your `node_modules`

See https://github.com/nativescript-vue/nativescript-vue-ui-tests/pull/6

**Comment by mtancoigne** [comment]
I created a repo with some configuration [for Appium and Nativescript-vue](https://github.com/el-cms/nativescript-vue-appium-example), maybe that can help some of you, and/or help for the later documentation.
